README.md

```diff
@@ -5,12 +5,12 @@
 Cucumber Screenplay is a small library for [Cucumber.js](https://github.com/cucumber/cucumber-js/) that enables better
```

Comment:
I am not sure why VSCode mangled this file so badly!

Reply:
Thanks for this @abendigo. Could you please undo all of the accidental edits here so your PR only contain relevant changes?

Reply:
Updated. Thanks @aslakhellesoy for taking a look! Turns out, I have "Format on Save" turned on at the user level.

Reply:
@aslakhellesoy Hey, any update on this?